src/OAuth/Manager.php

```diff
@@ -14,20 +16,34 @@ public function enabled()
 
     public function provider($provider)
     {
-        return static::$providers[$provider] = static::$providers[$provider] ?? new Provider($provider);
+        if (isset(static::$providers[$provider])) {
+            return static::$providers[$provider];
+        }
+
+        return $this->providers()->get($provider);
```

Comment:
Looks like you removed the setter. (`static::$providers[$provider] = ...`) so the guard here will now never return anything.